src/opennlp/tools/train.clj

```diff
@@ -53,26 +64,31 @@
   ([in] (train-treebank-chunker "en" in))
   ([lang in] (train-treebank-chunker lang in 100 5))
   ([lang in iter cut]
-     (with-open [rdr (reader in)]
-       (ChunkerME/train
-        lang
-        (ChunkSampleStream.
-         (PlainTextByLineStream. rdr))
-        cut iter))))
+    (ChunkerME/train
+      lang
+      (ChunkSampleStream.
+        (PlainTextByLineStream. 
+          (MarkableFileInputStreamFactory. (file in)) "UTF-8"))
```

Comment:
The reason I didn't wrap the (file in) calls is that, AFAIK, those are just wrappers for the java.io.File constructor, and a Java File object is basically just a wrapper for a filename -- it doesn't actually open a stream (i.e. no open() syscall takes place at that point). I believe management of the filedescriptor is encapsulated in OpenNLP's MarkableFileInputStreamFactory -- well, at least I hope it is :)